.github/workflows/test.yaml

```diff
@@ -11,3 +11,42 @@ jobs:
       self-hosted-runner: false
       self-hosted-runner-label: "edge"
       working-directory: ./bind-operator
+
+  django-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.2.2
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          cd dns-policy-app/app
+          python3 -m pip install --upgrade pip
+          pip install -r requirements.txt
```

Comment:
--no-cache-dir?

Reply:
I don't think that it makes a difference but I added it nonetheless